symbols.js

```diff
@@ -26,6 +26,7 @@ const common = {
   pilcrow2: '❡',
   pilcrow: '¶',
   plusMinus: '±',
+  question: '?',
```

Comment:
By the way: the `question` symbol was exactly the same between the `windows` and `other` sets, so I moved it to the `common` set.